src/css_parser.rs

```diff
@@ -2963,6 +2963,40 @@ mod css_tests {
             })));
     }
 
+    #[test]
+    fn test_parse_linear_gradient_5() {
+        assert_eq!(parse_css_background("linear-gradient(0.42rad, red, yellow)"),
+            Ok(StyleBackground::LinearGradient(LinearGradientPreInfo {
+                direction: Direction::Angle(FloatValue { number: 2375044 }),
```

Comment:
Use `FloatValue::new` here, otherwise the test is going to break if someone adjusts the precision on the floating point values. Same for the other test